charts/app-config/values-integration.yaml

```diff
@@ -1170,6 +1162,11 @@ govukApplications:
       sageMakerImage: 210287912431.dkr.ecr.eu-west-1.amazonaws.com/search
       sagemakerIamRole: arn:aws:iam::210287912431:role/learn-to-rank-sagemaker
       serviceAccountIamRole: arn:aws:iam::210287912431:role/search-api-learn-to-rank-govuk
+    contentStoreMongoPostgresCron:
+      mongoExport:
+        mongoDbUri: "{{ .Values.images.DraftContentStore.extraEnv.MONGODB_URI }}"
```

Comment:
This line seems odd to me on several levels :) Might be easiest to figure out in real-time with a brief chat if you have a minute?